<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>filterSearchWindow</class>
 <widget class="QDialog" name="filterSearchWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Przeglądanie</string>
  </property>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>330</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Anuluj</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLeftArrow">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>300</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>&lt;&lt;&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonRightArrow">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>300</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="text">
    <string>&gt;&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSearch">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Szukaj</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCID">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="placeholderText">
    <string>CID</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelName">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>401</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Imię</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelBoxNumber">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>401</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Box</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelHasHome">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>270</y>
     <width>431</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Ma dom?</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelAnimalType">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>401</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Rodzaj</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelFoundDate">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>150</y>
     <width>401</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Data</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelEyeColour">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>250</y>
     <width>401</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Kolor oczu</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSex">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>401</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Płeć</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelBreed">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>401</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Rasa</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelFoundDesantion">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>170</y>
     <width>401</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Miejsce</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelHairColour">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>230</y>
     <width>401</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Kolor sierści</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelHasDisease">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>401</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Chory?</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDiseaseName">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>210</y>
     <width>401</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Nazwa choroby</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelInfoName">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Imię: </string>
   </property>
  </widget>
  <widget class="QLabel" name="labelInfoAnimalType">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Rodzaj:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelInfoSex">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Płeć:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelInfoBoxNumber">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Box:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelInfoBreed">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Rasa:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelInfoFoundDate">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Data:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelInfoFoundStreet">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Miejsce:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelInfoHasDisease">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Chory?: </string>
   </property>
  </widget>
  <widget class="QLabel" name="labelInfoDiseaseName">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nazwa chroby: </string>
   </property>
  </widget>
  <widget class="QLabel" name="labelInfoHairColour">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Kolor sierści:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelInfoEyeColour">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Kolor oczu: </string>
   </property>
  </widget>
  <widget class="QLabel" name="labelInfoHasHome">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ma dom?:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
